### ISSUE TITLE
Import individual component in Nuxt 3

### ISSUE THREAD

**Issue by zerosdev** [bootstrap-vue/bootstrap-vue-next#963]
How to import individual component inside nuxt plugin instead of import all components by `modules` config ?

- OS: Windows 10
- Package Version: 0.6.7
- Npm/Yarn/Pnpm Version: yarn@1.22.19

**Comment by VividLemon** [comment]
The same way you'd import any component. Every component is accessible from the export object -- `import { BAlert } from bootsyrap-vue-next` there's also a complete "components" item, for convenience, but that import is probably not very treeshake friendly

**Comment by zerosdev** [comment]
I tried to create `bootstrap-vue.ts` inside `plugins` directory. Here is my code

```
import { BModal } from "bootstrap-vue-next"
export default defineNuxtPlugin((nuxtApp) => {
    nuxtApp.vueApp.component('BModal', BModal)
})
```

But there is an error `document is not defined`
I think that is because i'm using universal mode which mean it's running in SSR mode but why it can work properly when using `modules` config

**Comment by VividLemon** [comment]
No, its a known nuxt compatibility issue. Hypothetically one could use client only mode, but that defeats the purpose of nuxt... I suppose running client only could be a good temporary solution to those wanting to use nuxt, but can't wait

**Comment by VividLemon** [comment]
Will be fixed in 0.8
